lnd_test.go

```diff
@@ -276,6 +276,58 @@ func closeChannelAndAssert(ctx context.Context, t *harnessTest,
 	return closingTxid
 }
 
+// waitForChannelPendingForceClose waits for the node to report that the
```

Comment:
This commit can be deleted when rebasing to master as it was merged on its own.

Reply:
Deleted.